content/ru/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,186 @@
+```json
+{
+   "ts": 1580306777.04728,
+   "v": 4,
+   "msg": "Pod status updated",
+   "pod":{
+      "name": "nginx-1",
+      "namespace": "default"
+   },
+   "status": "ready"
+}
+```
+
+Специальные ключи:
+* `ts` — временная метка в формате времени Unix (обязательный параметр, float);
+* `v` — детализация (для общей информации — не для сообщений об ошибках, int);
+* `err` — ошибка (опциональный параметр, string);
+* `msg` — сообщение (обязательный параметр, string).
+
+
+Список компонентов, поддерживающих формат JSON:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+* {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}}
+* {{< glossary_tooltip term_id="kube-scheduler" text="kube-scheduler" >}}
+* {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
+
+### Уровень детализации лога
+
+Флаг `-v` задает степень детализации лога. Увеличение значения увеличивает количество регистрируемых событий. Уменьшение значения уменьшает количество регистрируемых событий. Увеличение детализации приводит к тому, что регистрируются все менее значимые события. При уровне детализации, равном 0, регистрируются только критические события.
+
+### Местоположение лога
+
+Существует два типа системных компонентов: те, которые работают в контейнере, и те, которые работают за пределами контейнера. Например:
+
+* Планировщик Kubernetes и kube-proxy работают в контейнере.
+* kubelet и {{<glossary_tooltip term_id="container-runtime" text="среда исполнения для контейнеров">}}
+  работают за пределами контейнеров.
+
+На машинах с systemd среда исполнения и kubelet пишут в journald. В противном случае ведется запись в файлы `.log` в директории `/var/log`. Системные компоненты внутри контейнеров всегда пишут в файлы `.log` в директории `/var/log`, обходя механизм логирования по умолчанию. как и логи контейнеров, логи системных компонентов в `/var/log` нуждаются в ротации. В кластерах Kubernetes, созданных с помощью скрипта `kube-up.sh`, ротация логов настраивается с помощью инструмента `logrotate`. `logrotate` ротирует логи ежедневно или при достижении ими размера в 100 МБ.
+
+## {{% heading "whatsnext" %}}
+
+* [Архитектура логирования в Kubernetes](/docs/concepts/cluster-administration/logging/)
+* [Структурированное логирование](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)
+* [Контекстное логирование](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging)
+* [Вывод флагов klog из эксплуатации](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
+* [Соглашения и правила для определения критичности логов](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)
```

Comment:
Optionally, mark these 4 links as being to English pages.

Reply:
Thanks for the suggestion!